docs/console/project-configuration/create-a-project.mdx

```diff
@@ -59,24 +59,26 @@ The Project creation process is divided into three phases:
 
     If there are no configured Git Providers for the company, an informative modal is displayed, and you will not be able to continue.
 
-   * **Git repo path**: the path, calculated from Project name and company, where the configuration will be saved on the Git Provider. It specifies the location of your Project on the Git Provider website;
+   * **Git repo path**: the path where the configuration will be saved on the Git Provider. It specifies the location of your Project on the Git Provider website;
```

Comment:
è corretto il nome? lato FE non mostriamo Repository path template?

Reply:
`Git repo path` is the correct name  for the field. Maybe you are confusing it with the one on the provider form.